redirects.json

```diff
@@ -0,0 +1,508 @@
+	  "value":"/builders/interoperability/xcm/remote-execution/remote-evm-calls/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/xc20/mintable-xc20/",
+	  "value":"/builders/interoperability/xcm/xc20/overview/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/xcm-transactor/",
+	  "value":"/builders/interoperability/xcm/remote-execution/substrate-calls/"
+	},
+	{
+	  "key":"/builders/interoperability/xcm/xc-integration/",
+	  "value":"/builders/interoperability/xcm/xc-registration/"
+	},
+	{
+	  "key":"/builders/pallets-precompiles/precompiles/xcm-utils/",
+	  "value":"/builders/interoperability/xcm/xcm-utils/"
+	},
+	{
+	  "key":"/builders/xcm/fees/",
+	  "value":"/builders/interoperability/xcm/core-concepts/weights-fees/"
+	},
+	{
+	  "key":"/builders/xcm/xc20/index/",
+	  "value":"/builders/interoperability/xcm/xc20/"
+	},
+	{
+	  "key":"/builders/xcm/xc20/overview/",
+	  "value":"/builders/interoperability/xcm/xc20/overview/"
+	},
+	{
+	  "key":"/builders/xcm/xc20/xtokens/",
+	  "value":"/builders/interoperability/xcm/xc20/send-xc20s/"
+	},
+	{
+	  "key":"/builders/xcm/xcm-sdk/xcm-sdk/",
+	  "value":"/https:/moonbeam-foundation.github.io/xcm-sdk/latest/"
+	},
+	{
+	  "key":"/builders/xcm/xcm-transactor/",
+	  "value":"/builders/interoperability/xcm/remote-execution/substrate-calls/"
+	},
+	{
+	  "key":"/builders/xcm/xc-integration/",
+	  "value":"/builders/interoperability/xcm/xc-registration/"
+	},
+	{
+	  "key":"/getting-started/using-metamask/",
+	  "value":"/tokens/connect/metamask/"
+	},
+	{
+	  "key":"/getting-started/using-remix/",
+	  "value":"/builders/ethereum/dev-env/remix/"
+	},
+	{
+	  "key":"/getting-started/using-truffle/",
+	  "value":"/builders/ethereum/dev-env/"
+	},
+	{
+	  "key":"/getting-started/local-node/deploy-contract/",
+	  "value":"/builders/ethereum/libraries/"
+	},
+	{
+	  "key":"/getting-started/local-node/using-remix/",
+	  "value":"/builders/ethereum/dev-env/remix/"
+	},
+	{
+	  "key":"/getting-started/local-node/web3-transaction/",
+	  "value":"/builders/ethereum/libraries/web3py/"
+	},
+	{
+	  "key":"/getting-started/moonriver/integrate-metamask/",
+	  "value":"/builders/get-started/networks/moonriver/"
+	},
+	{
+	  "key":"/getting-started/testnet/faucet/",
+	  "value":"/builders/get-started/networks/moonbase/"
+	},
+	{
+	  "key":"/getting-started/testnet/metamask/",
+	  "value":"/tokens/connect/metamask/"
+	},
+	{
+	  "key":"/getting-started/testnet/precompiles/",
+	  "value":"/builders/ethereum/precompiles/"
+	},
+	{
+	  "key":"/getting-started/testnet/pubsub/",
+	  "value":"/builders/ethereum/json-rpc/pubsub/"
+	},
+	{
+	  "key":"/integrations/openzeppelin/overview/",
+	  "value":"/builders/ethereum/dev-env/openzeppelin/overview/"
+	},
+	{
+	  "key":"/learn/dapps-list/dtmb/",
+	  "value":"/learn/dapps-list/"
+	},
+	{
+	  "key":"/learn/features/unified-accounts/",
+	  "value":"/learn/core-concepts/unified-accounts/"
+	},
+	{
+	  "key":"/learn/platform/roadmap/",
+	  "value":"//"
```

Comment:
Is this supposed to be a double-slash?

Reply:
good catch! Actually no! This redirect is broken